docs/healpix.rst

```diff
@@ -0,0 +1,55 @@
+**************
+HEALPIX images
```

Comment:
Yes, I should clarify, and I think we could allow more fine-tuned control over the order of the interpolation.